ScoutSuite/providers/azure/rules/findings/graphrbac-guest-users.json

```diff
@@ -0,0 +1,11 @@
+{
+    "description": "Guest Users in Use",
+	"rationale": "<b>Description:</b><br><br>Avoid creating guest users, as they are typically added outside your employee on-boarding/off-boarding process and could potentially be overlooked indefinitely leading to a potential vulnerability.<br><br><b>References:</b><ul><li>CIS Microsoft Azure Foundations v1.1.0 1.3</li></ul>",
+    "dashboard_name": "Users",
+    "path": "graphrbac.users.id",
+    "display_path": "graphrbac.users.id",
+    "conditions": [ "and",
+        [ "graphrbac.users.id.user_type", "equal", "Guest" ]
+    ],
+    "id_suffix": "user_type"
```

Comment:
This should be set to the last suffix in the HTML object's ID so that it can be highlighted.

Reply:
Which in this case is `user_type` (I changed it from the value you set).